src/braket_container.py

```diff
@@ -50,9 +51,20 @@ def log_failure_and_exit(*args):
     with open(ERROR_LOG_FILE, 'a') as error_log:
         for text in args:
             error_log.write(text)
-            print(text)
-    sys.exit(0)
```

Comment:
Ugh. I forgot about this. We have to exit with exit code 0 otherwise SageMaker will not propagate our error correctly. Will we ensure we exit with 0 in all code paths?

Reply:
Done